src/components/dark-theme-button/components/moon-fill-icon/moon-fill-icon.tsx

```diff
@@ -1,34 +1,35 @@
 import * as React from 'react'
 
 import { motion } from 'framer-motion'
+import { MOTION_TRANSITION, MOTION_WHILE_TYPE_VARIANT } from '~/lib/constants/motion'
 
-import { MOTION_TRANSITION } from '~/lib/constants/motion'
-
-interface MonnFillIconProps {
+interface MoonFillIconProps {
   className?: string
 }
 
-const coreVariants = {
-  initial: { opacity: 0, rotate: -90, y: 10 },
-  animate: { opacity: 1, origin: 'bottom', rotate: -0, y: 0, transition: MOTION_TRANSITION },
+const variants = {
+  initial: { opacity: 0, scale: 0.8, originX: 'center', originY: 'center', rotate: -80 },
+  animate: { opacity: 1, scale: 1, rotate: 0, transition: MOTION_TRANSITION },
 }
 
-export function MoonFillIcon(props: MonnFillIconProps) {
+export function MoonFillIcon(props: MoonFillIconProps) {
   const { className } = props
 
   return (
     <motion.svg
-      className={className}
-      width="16px"
-      height="16px"
-      fill="currentColor"
-      viewBox="0 0 20 20"
+      className={`w-full ${className}`}
```

Comment:
I recommend using a utility for constructing className strings conditionally.

for do that, you can use [clsx](https://www.npmjs.com/package/clsx)

```suggestion
      className={clsx(`w-full`, className)}
```

Reply:
great, already changed, thanks 🎉